sky/utils/env_options.py

```diff
@@ -2,6 +2,8 @@
 import enum
 import os
 
+SPOT_CONTROLLER_CLOUD_ENV_VAR = 'SKYPILOT_SPOT_CONTROLLER_CLOUD'
```

Comment:
How about move it to https://github.com/skypilot-org/skypilot/blob/84d3607d4bb59d82dd6aca70afbc2a9328ee2b99/sky/spot/constants.py and mark `Internal: for testing only; the cloud to use for a spot controller if it does not exist.`?

Reply:
I am trying to keep all the environment variable in the same place for better visibility, and this env var is not purely for internal purpose. Wdyt?